scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -340,6 +344,25 @@ class RedundantBraces(ftoks: FormatTokens) extends FormatTokensRewrite.Rule {
   ): Boolean =
     rhs.eq(b) && getSingleStatIfLineSpanOk(b).exists(innerOk(b))
 
+  private def defnBodies(
```

Comment:
```suggestion
  private def shouldRemoveDefnBraces(
```
maybe a bit better explaining ?

Reply:
ok, will explain